debian/changelog

```diff
@@ -1,3 +1,52 @@
+ubuntu-advantage-tools (27.5~22.04.1) jammy; urgency=medium
+
+  * d/control:
+    - Update homepage URL
+  * d/rules:
+    - remove calls to enable/start the license-check timer
+    - add calls to enable/start the ua daemon
+  * d/tools.postinst:
+    - remove logic to trigger license-check
+    - purge old license-check.timer/service/path files
+  * d/tools.postrm:
+    - include the daemon log file in the list of logs to remove
+    - use common prefix to remove ua related gpg files
+  * New upstream release 27.5:
+    - aws: add support for the IPv6 metadata endpoint
+    - cis: update URL for the documentation
+    - cli:
+      + add endpoint to simulate the status using a specific token
+      + fix return code when attaching an already attached machine (GH: #1867)
+      + fix security-status to consider all possible origins to show updates
+      + include cloud build.info in the collect-logs tarball
+      + only show services which exist in the contracts server in ua status
+    - clouds: add pro license check methods
```

Comment:
This entry is related to the daemon work right ? If yes, I think we should exclude it